presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -66,6 +69,8 @@
     private Duration queryMaxExecutionTime = new Duration(100, TimeUnit.DAYS);
     private Duration queryMaxCpuTime = new Duration(1_000_000_000, TimeUnit.DAYS);
 
+    private DataSize queryMaxScanPhysicalBytes = DataSize.succinctDataSize(1, PETABYTE);
```

Comment:
Is `QueryStats::rawInputDataSize` equivalent to scanPhysicalBytes?  does reading from materialized intermediate table included? cc @arhimondr

Reply:
Same comment as in `SystemSessionProperties` - lets use the same name as the actual metric - `queryMaxRawInputBytes`. In accordance with that lets change the config name as well - `query.max-raw-input-bytes`.